### ISSUE TITLE
authenticate failed when other process listening port 80

### ISSUE THREAD

**Issue by miaulightouch** [aleab/toastify#118]
If this is a crash, bug or unexpected behaviour report, please use the following template.

## EXPECTED BEHAVIOUR
browser should pass the token to toastify.

## ACTUAL BEHAVIOUR
port 80 is listened by other process, toastify would never get the token.

## STEPS TO REPRODUCE THE BEHAVIOUR
1. start a web server and listen on 80.
2. start toastify and auth.

## LOG FILE
*(Drag and drop your `Toastify.log` file here; it can be found in `%LocalAppData%\Toastify`)*

**Comment by aleab** [comment]
Will be fixed in the next release.
2772e4d

**Comment by warizzmann** [comment]
also this makes me unable to run my apache (i use XAMPP) because the port is being used by toastify.

**Comment by PedroS11** [comment]
Yeah, I have the same problem as @warizzmann. My .Net core API uses port 80 and I needed to change that.
